packages/tasks/src/model-libraries.ts

```diff
@@ -426,6 +426,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/DigitalPhonetics/IMS-Toucan",
 		countDownloads: `path:"embedding_gan.pt" OR path:"Vocoder.pt" OR path:"ToucanTTS.pt"`,
 	},
+	"index-tts": {
+		prettyLabel: "IndexTTS",
+		repoName: "IndexTTS",
+		repoUrl: "https://github.com/index-tts/index-tts",
+		snippets: snippets.indextts,
+		filter: false,
```

Comment:
```suggestion
```

Default value, I think.

Reply:
but we put `false` for all of the ones we don't want so prolly okay to keep it the way it is, unless we want to make the change for all.

Reply:
We have not been consistent AFAIK, but ok.

Reply:
yeah to be honest I tend to approve both with or without explicit `filter: false` without caring much. Happy to review a follow-up PR to harmonize this (removing all unecessary `filter: false`) but low prio